website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -1,197 +1,173 @@
+| -------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------ |
+| **Update Mode**              | Formatting  | Controls the save experience of an edited cell.<br/> **Single row** - Cells can be saved using the Save/Discard column buttons. <br/> **Multi row** - cells can be saved by using an **onSubmit** event of the column or through an external button widget. | NA                             |
+| **Cell Wrapping**            | Formatting  | Controls how overflowing contents of the column are handled.<br/> **on** - Contents get wrapped to the next line.<br/> **off** - Contents are truncated with an ellipsis (...). | NA |
+| **Editable**                 | Formatting  | Controls whether cells of the column are editable                                                             | NA                             |
+| **Regex**                    | Validation  | Enter a regular expression that user input must match to be considered valid. Displays an error message on failure.                                   | NA                             |
+| **Valid**                    | Validation  | Enter a JS expression that evaluates whether the user's input is valid.                                                                | NA                             |
+| **Error Message**            | Validation  | The error message displays if the input fails the **Regex** or **Valid** properties' checks.                                        | NA                             |
+| **Required**                 | Validation  | Makes input to the widget mandatory.                                                                          | NA                             |
+| **Min**                      | Validation  | Sets the minimum allowed value.                                                                               | NA                             |
+| **Max**                      | Validation  | Sets the maximum allowed value.                                                                               | NA                             |
+| **updatedRows**              | Binding     | Contains all the data of the edited table rows.                                                    | `{{Table1.updatedRows}}`       |
+| **updatedRowIndices**        | Binding     | Contains an array of indices of the table rows that have been edited.                                                        | `{{Table1.updatedRowIndices}}` |
+| **updatedRow**               | Binding     | Contains the data of the row that triggered the **onSubmit**, **onSave** or **onDiscard** action. | `{{Table1.updatedRow}}`|
 
-![](</img/Update_mode.png>)
+#### Update mode (single row)
```

Comment:
@jnikhila 

I like this idea (it's what I attempted originally), however I ran into the issue of the size/appearance of the headings on the page. In the above strategy, "Single-row" and "Multi-row" would both be 5th level headings (#####), which appear _smaller on the page_ than even regular bold text.

For example, 4th level headings are already the same size as bold text:

![as_bold](https://user-images.githubusercontent.com/51402465/203237556-8451cc91-604b-44cd-9919-6b1ef6eb8163.png)

I tried just using the bold text alone to separate the subheadings, but since each of those mode sections include guides, steps, pictures, etc -- It was very hard to parse and find the separation between them. (Which is why there's still bold text _and_ subheadings, to help make it as easy as possible to find the border btwn them).

Let me know what you think, maybe there's a better solution.

Reply:
For such headings we can add our own styles to change the default behaviour of Docusaurus. Do you think, you can give it a go and see if you could do anything here?

Reply:
Sure! I added css rules for `h1` thru `h5`, setting `font-size` values to steps between `3rem` and `1.25rem`.

Net effect:
- `h1` page titles remain the same size, and all other subheadings `h2` - `h5` are a little bit larger.
- `h5` headings are larger than regular bold text
- `h4` headings (almost all of our detailed properties descriptions below properties tables are `h4`) are easier to pick out from surrounding content, which I think is nice.

Reply:
Awesome. @danciaclara - Do you want to take a look and update?